src/sdk.js

```diff
@@ -122,9 +124,15 @@ class OpenForm {
               languageSelectorTarget={this.languageSelectorTarget}
               onLanguageChangeDone={this.onLanguageChangeDone.bind(this)}
             >
-              <Router basename={this.basePath}>
-                <App form={this.formObject} />
-              </Router>
+              {this.useHashRouting ? (
+                <HashRouter basename={this.basePath}>
+                  <App form={this.formObject} />
+                </HashRouter>
+              ) : (
+                <BrowserRouter basename={this.basePath}>
+                  <App form={this.formObject} />
+                </BrowserRouter>
+              )}
```

Comment:
you can also make the component dynamic:

```jsx
const Router = this.useHashRouting ? HashRouter : BrowserRouter;
...
<Router basename={this.basePath><App ... /></Router>
```
so then the actual router wrapper code is not modified, just the component alias is.

Reply:
Heh, I had this but changed it to make it more clear/specific. I'll change to this.